src/frontend/Sider/Oppgavebenk/Oppgaveknapp.tsx

```diff
@@ -61,9 +67,20 @@ const Oppgaveknapp: React.FC<Props> = ({ oppgave }) => {
             .finally(() => settLaster(false));
     };
 
+    const gåTilJournalføring = (type: 'klage' | 'stønad') => {
+        const journalpostId = oppgave.journalpostId || '';
+        const oppgaveId = oppgave.id || '';
+        const gjelderKlage = type === 'klage';
+        navigate(lagJournalføringUrl(journalpostId, oppgaveId, gjelderKlage));
+    };
+
     const gåTilOppgaveUtførelse = () => {
         if (oppgaveErSaksbehandling(oppgave)) {
             gåTilBehandleSakOppgave();
+        } else if (oppgaveErJournalføringKlage(oppgave)) {
+            gåTilJournalføring('klage');
+        } else if (oppgaveErJournalføring(oppgave)) {
```

Comment:
Skal det legges til journalføring for klage enda? Vi har jo ikke klage atm så er det vits å ha en "side" for det?

Reply:
Dere skal håndtere journalføring av klage fra den samme siden som man ellers bruker, så denne kan fjernes ja :) 👍

Reply:
Utbedret